README.md

```diff
@@ -7,68 +7,49 @@
 
 # Knative Source - MongoDB
 
-## Development
-
-### Dependencies
-
-1. [go](https://golang.org/doc/install)
-
-### Setup environment
-
-Put the following in a `./bashrc` or `./bashprofile`
-
-```sh
-export GOPATH="$HOME/go"
-export PATH="${PATH}:${GOPATH}/bin"
-```
-
-### Clone to your machine
-
-1. [Fork this repo](https://help.github.com/articles/fork-a-repo/) to your account
-2. Clone to your machine
-
-```sh
-mkdir -p ${GOPATH}/src/github.com/googleinterns
-cd ${GOPATH}/src/github.com/googleinterns
-git clone git@github.com:${YOUR_GITHUB_USERNAME}/knative-source-mongodb.git
-cd knative-source-mongodb
-git remote add upstream https://github.com/googleinterns/knative-source-mongodb.git
-git remote set-url --push upstream no_push
-```
-
-`Upstream` will allow for you to [sync your fork](https://help.github.com/articles/syncing-a-fork/)
-
-### Building
-
-`go build ./...`
-
-### Running unit tests
-
-`go test ./...`
-
-### Updating dependencies
-
-`./hack/update-deps.sh`
-
-## Source Code Headers
-
-Every file containing source code must include copyright and license
-information. This includes any JS/CSS files that you might be serving out to
-browsers. (This is to help well-intentioned people avoid accidental copying that
-doesn't comply with the license.)
-
-Apache header:
-
-    Copyright 2020 Google LLC
-
-    Licensed under the Apache License, Version 2.0 (the "License");
-    you may not use this file except in compliance with the License.
-    You may obtain a copy of the License at
-
-        https://www.apache.org/licenses/LICENSE-2.0
-
-    Unless required by applicable law or agreed to in writing, software
-    distributed under the License is distributed on an "AS IS" BASIS,
-    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-    See the License for the specific language governing permissions and
-    limitations under the License.
+The MongoDb Event source adds support of MongoDb ressources to Knative Eventing.
+
+## Prerequisites
+
+1. Install [Knative Eventing](https://knative.dev/docs/install/any-kubernetes-cluster/#installing-the-eventing-component) in your Kubernetes Cluster.
+
+2. Install MongoDb and create a Replica Set using the following [instructions](https://www.mongodb.com/blog/post/running-mongodb-ops-manager-in-kubernetes#:~:text=The%20MongoDB%20Enterprise%20Kubernetes%20Operator%2C%20or%20simply%20the%20Operator%2C%20manages,changing%20these%20settings%20as%20needed).
```

Comment:
nit: I'd suggest saying something like. `Install MongoDb and create a Replica Set. You can use the following instructions

just in case folks want to install it from wherever, there is no need for them to follow this particular instructions. We just give a suggestion

Reply:
True, Ill change!